control-plane/agents/src/bin/core/controller/policies/rebuild_policies.rs

```diff
@@ -50,27 +48,23 @@ impl RuleSet {
             return cli_twait;
         }
 
-        let _pol = Self::assign(nexus.size);
-        Self::default_faulted_child_timewait()
+        let pol = Self::assign(nexus.size);
 
-        // TODO: Enable when policies are dynamic
-        /* match pol {
+        match pol {
             RuleSet::SystemAvail(pol) => pol.faulted_child_wait_duration(nexus),
             RuleSet::SystemPerf(pol) => pol.faulted_child_wait_duration(nexus),
-        } */
+        }
     }
 
     /// Assign a rebuild policy to the nexus.
-    fn assign(size: u64) -> Self {
+    fn assign(_size: u64) -> Self {
+        RuleSet::SystemPerf(SystemPerf {})
+        /*
```

Comment:
shall we remove this commented out code?

Reply:
This will be required at some point, it can remain harmlessly here I guess.

Reply:
It's harmless but it's also pointless, we use git anyway so it's in the history anyway if you want to recover it